<!--yml

category: 未分类

date: 2024-05-29 12:38:31

-->

# GitHub - failsafe-go/failsafe-go: 用于 Go 的容错和恢复力模式

> 来源：[https://github.com/failsafe-go/failsafe-go](https://github.com/failsafe-go/failsafe-go)

<include-fragment class="js-notification-shelf-include-fragment" data-base-src="https://github.com/notifications/beta/shelf"></include-fragment>

<main id="js-repo-pjax-container"> <turbo-frame id="repo-content-turbo-frame" target="_top" data-turbo-action="advance" class=""># failsafe-go/failsafe-go

此提交不属于存储库的任何分支，并且可能属于存储库之外的分支。

<include-fragment src="/failsafe-go/failsafe-go/spoofed_commit_check/34e615d42ac241c40f224a07458914299a6357ed" data-test-selector="spoofed-commit-check"></include-fragment></turbo-frame></main>

<ghcc-consent id="ghcc" class="position-fixed bottom-0 left-0" data-initial-cookie-consent-allowed="" data-cookie-consent-required="false">您目前无法执行此操作。

<template id="site-details-dialog"></template><template id="snippet-clipboard-copy-button"></template><template id="snippet-clipboard-copy-button-unpositioned"></template></ghcc-consent>
